.github/workflows/linux-eic-shell.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         run: |
           # install this repo
           CC="${{ matrix.CC }}" CXX="${{ matrix.CXX }}" CXXFLAGS="${{ matrix.CXXFLAGS }}" cmake -B build -S . -DCMAKE_INSTALL_PREFIX=install -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_BUILD_TYPE=${{ matrix.CMAKE_BUILD_TYPE }} -DUSE_ASAN=ON -DUSE_TSAN=OFF -DUSE_UBSAN=ON
-          cmake --build build -- -j 2 install
+          cmake --build build -j $(getconf _NPROCESSORS_ONLN) --target install
```

Comment:
I was going to suggest `nproc` so I looked up what the benefits and drawbacks are.

`nproc`: Not POSIX and needs coreutils. Returns number of processors available to calling process, regardless of physical processors (I guess this accounts for cgroup restrictions).

`getconf`: POSIX command but non-POSIX config key. More portable since not just Linux. Returns physical cores, not accounting for hyper-threading and (I presume, again) cgroups.

As long as it returns 4, it doesn't matter which one to me.

Reply:
From my experiments, _NPROCESSORS_ONLN practically gives logical cores, not physical. And, yes, works on macOS.